### DISCUSSION TITLE
3.0.2版本，下载最后的几个分片速度很慢

### DISCUSSION THREAD

**Discussion by shellcmd89** [nilaoda/N_m3u8DL-CLI#712] [Q&A] [answered]
我下载日剧较多，有350或者800-1100各种分片数。我记得以前的版本从下载首分片开始速度一直保持几MB/s，直到结束都差不多。升级到3.0.2版本以后下载到最后10个左右分片的时候，速度变得很慢，只有16-几百kB/s不等，虽然最后也能下载完成，但是会额外的多消耗好几分钟的时间。请问有没有同学遇到和我一样情况的？

**Comment by nilaoda** [comment, marked as answer]
```
--stopSpeed                当速度低于此值时，重试(单位为KB/s)
```

**Reply by shellcmd89** [reply]
感谢您的回复，我试试添加这个参数。
